src/util/nodeSelectors.ts

```diff
@@ -40,11 +40,11 @@ export function selectWithLeadingDelimiter(
   editor: TextEditor,
   node: SyntaxNode
 ): SelectionWithContext {
-  const leadingDelimiterToken = node.previousSibling;
+  const leadingNonDelimiterToken = node.previousSibling?.previousSibling;
```

Comment:
Note that we now use the end of the token before the delimiter rather than the start of the delimiter.  This way, for example, if you have "foo = bar" and say "chuck value", you won't get the trailing space after `foo`.  See changed test case above (`chuckValue`)